brands/shop/beverages.json

```
@@ -135,5 +135,15 @@
       "name": "茶湯會",
       "shop": "beverages"
     }
+  },
+  "shop/beverages|Sagasser":{
+    "countryCodes": ["de"],
+    "tags": {
+      "brand": "Sagasser",
+      "brand:wikidata":"Q30838417" ,
+      "brand:wikipedia": "de:Sagasser", 
+      "name": "SAGASSER-Vertriebs GmbH",
```

Just "Sagasser" is fine here..
```suggestion
      "name": "Sagasser",
```